generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -20,7 +20,7 @@
     EddyCurrentProxProbeSensitivityUnits, ExcitationDCorAC,
```

Comment:
For each of these PRs, I would expect the existing source file to be deleted. So in this case `nidaqmx/_task_modules/channels/ai_channel.py`.

Reply:
OK, disregard this. See below.